src/components/common/ListPeopleAndRoles.vue

```diff
@@ -34,33 +34,27 @@
           :key="index"
           no-gutters
         >
-          <v-col class="name-column text-truncate">
-            <v-icon v-if="isPerson(orgPerson)">mdi-account</v-icon>
-            <v-icon v-if="isOrg(orgPerson)">mdi-domain</v-icon>
-            <v-tooltip
-              top
-              :disabled="formatName(orgPerson).length < 25"
-              content-class="top-tooltip"
-              transition="fade-transition"
-            >
-              <template v-slot:activator="{ on }">
-                <span v-on="on" class="name ml-2">{{ formatName(orgPerson) }}</span>
-              </template>
-              <span>{{ formatName(orgPerson) }}</span>
-            </v-tooltip>
-
-            <div v-if="officerEmail(orgPerson)" class="text-wrap">
-              <p class="ml-8">{{ officerEmail(orgPerson) }}</p>
+          <v-col class="name-column d-flex">
```

Comment:
`d-flex` makes the child divs flexbox containers (kind of like v-cols) so they line up nicely. This is so the icon is in "column 1" and the other stuff in the next "column".

![image](https://user-images.githubusercontent.com/10002889/160507733-f8348d53-2e46-493d-a633-2ee7f76600b0.png)